### DISCUSSION TITLE
Galaxy Brain 뱃지 받을 수 있게 채택 받을 수 있나요?

### DISCUSSION THREAD

**Discussion by YuHyeonWook** [learningssik/pull-request#120] [General]
- 제목 그대로 입니다.
- For zisuzin 2, For sh5124y 2에 답변 달았습니다!

**Comment by learningssik** [comment]
해당 글은 다른 분을 위해 만든거라서 #121  #122 에 답변다시면 채택드리겠습니다! 감사합니다.
